packages/toolpad-utils/src/collections.ts

```diff
@@ -93,3 +95,66 @@ export function filterKeys<U>(
 ): Record<string, U> {
   return mapProperties(obj, ([key, value]) => (filter(key) ? [key, value] : null));
 }
+
+/**
+ * Check if two values are equal - that is,
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isReferEqual(obj: any, objToCompare: any): boolean {
```

Comment:
Would it be possible to use `unknown` types instead?

Reply:
![image](https://github.com/mui/mui-toolpad/assets/27261522/dcf903b1-6129-44bc-96ad-6fae3e465c27)
I tried this way but unfortunately, these codes don’t work. And I found a lot of materials about this that all use the any

Reply:
![image](https://github.com/mui/mui-toolpad/assets/27261522/119f4bf8-1666-4d6f-a860-31f43eb8f4b4)

Reply:
I fixed it